src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -140,8 +146,22 @@
   }
 
   private static String getGapicVersion() {
-    String packageVersion = {@xsettingsClass.name}.class.getPackage().getImplementationVersion();
-    return packageVersion != null ? packageVersion : DEFAULT_GAPIC_VERSION;
+    if (gapicVersion == null) {
+      try {
+        Properties properties = new Properties();
+        String mavenPropertiesPath =
+            META_FILE_ROOT + GROUP_ID + "/" + ARTIFACT_ID + "/pom.properties";
+        InputStream inputStream =
+            ImageAnnotatorSettings.class.getResourceAsStream(mavenPropertiesPath);
```

Comment:
You have a fixed class name in here (`ImageAnnotatorSettings`) - please run the tests in api-client-staging with your generated code before submitting.